rslib/src/sync/http_server/mod.rs

```diff
@@ -219,7 +219,10 @@ impl SimpleServer {
             }
         }
     }
-
+    pub fn is_running() -> bool {
+        let addr = "127.0.0.1:8080";
```

Comment:
Should not hardcode host and port.

Reply:
will use `let config = envy::prefixed("SYNC_")
            .from_env::<SyncServerConfig>().unwrap();` to grab host and port

Reply:
<img width="863" alt="image" src="https://github.com/user-attachments/assets/19e81326-4ec6-41b1-8223-8bad3aed2f39">